yarn.lock

```diff
@@ -40,20 +40,20 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
-"@babel/core@^7.0.0":
```

Comment:
This lockfile got changed but our dependencies were not changed on this branch. Is there a reason to include these changes on this PR?

Reply:
Nevermind, just saw your comment about publicPath support